pydantic_ai_slim/pydantic_ai/models/anthropic.py

```diff
@@ -196,7 +196,7 @@ async def _messages_create(
         anthropic_messages: list[MessageParam] = []
 
         for m in messages:
-            if m.role == 'system':
+            if m.message_kind == 'system-prompt':
```

Comment:
probably better to us `isinstance`. @dmontagu already changed it in most places.

Reply:
Good idea, will do in a different PR.